tests/node_test.cc

```diff
@@ -776,6 +778,21 @@ TEST_CASE("Node is checked for 2D case", "[node][2D]") {
       for (unsigned i = 0; i < velocity.size(); ++i)
         REQUIRE(node->velocity(Nphase)(i) ==
                 Approx(velocity(i)).epsilon(Tolerance));
+
+      // Apply rotation matrix with Euler angles alpha = 10 deg, beta = 30 deg
```

Comment:
Could you create an independent test for general velocity constraints? this test case doesn't show what constraint is being applied? Could you prove that in the specified direction the constraint velocity is zero